fictadvisor-api/src/v2/api/controllers/PermissionController.ts

```diff
@@ -13,31 +13,23 @@ import { JwtGuard } from '../../security/JwtGuard';
 })
 export class PermissionController {
   constructor (
-      private permissionService: PermissionService,
+    private permissionService: PermissionService,
   ) {}
 
-  @ApiOkResponse({
-    type: CheckPermissionsResponse,
-  })
-  @ApiBadRequestResponse({
-    description: `\n      
-    DataNotFoundException:
-      Data were not found
-      
-    InvalidBodyException:
-      obj.permissions: permissions must be an array
-      obj.permissions: each value in permissions must be one of the following values`,
-  })
   @ApiEndpoint({
     summary: 'Validates user access for specified permissions',
+    documentation: PermissionDocumentation.CHECK,
```

Comment:
Name of the documentation property (CHECK) must be the same as corresponding controller method (CHECK_PERMISSIONS)